GUI/src/cz/fidentis/gui/comparison_batch/BatchComparisonResults.java

```diff
@@ -1980,6 +2017,11 @@ private void selectionButtonActionPerformed(java.awt.event.ActionEvent evt) {//G
             getContext().getHDinfo().setsType(SelectionType.ELLIPSE);
         }
         GUIController.getSelectedProjectTopComponent().getViewerPanel_Batch().setSelection(selectionButton.isSelected());
+        if (selectionButton.isSelected()){
```

Comment:
dalo by sa to zjednodusit len na jeden riadok?
`jButtonLocalAreas.setEnabled(!selectionButton.isSelected());`

Reply:
ok